tests/src/main/java/net/neoforged/neoforge/debug/capabilities/handlers/energy/EnergyItemTests.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.debug.capabilities.handlers.energy;
+
+import net.minecraft.util.Mth;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.neoforged.neoforge.capabilities.Capabilities.EnergyHandler;
+import net.neoforged.neoforge.transfer.EnergyHandlerUtil;
+import net.neoforged.neoforge.transfer.handlers.IItemContext;
+import net.neoforged.neoforge.transfer.handlers.templates.contexts.PlayerItemContext;
+import net.neoforged.neoforge.transfer.handlers.templates.energy.EnergyBufferAttachment;
+import net.neoforged.neoforge.transfer.transaction.Transaction;
+import net.neoforged.neoforge.transfer.transaction.TransactionContext;
+import net.neoforged.testframework.annotation.ForEachTest;
+import net.neoforged.testframework.annotation.TestHolder;
+import net.neoforged.testframework.gametest.EmptyTemplate;
+import net.neoforged.testframework.gametest.ExtendedGameTestHelper;
+import net.neoforged.testframework.gametest.GameTest;
+
+@ForEachTest(groups = EnergyTestsSetup.GROUP_ID, idPrefix = "handler.energy.item.")
+public class EnergyItemTests {
```

Comment:
Please move as many tests as you can (i.e. when level access is not needed) to junit tests.